app/views/api/v1/conversations/partials/_conversation.json.jbuilder

```diff
@@ -17,7 +17,7 @@ json.meta do
 end
 
 json.id conversation.display_id
-if conversation.messages.count.zero?
+if conversation.message_count.zero?
   json.messages []
 elsif conversation.unread_incoming_messages.count.zero?
```

Comment:
@scmmishra Can we preload this as well?

Reply:
I managed to do it, however in `models/messages.rb`, it is calculated again in `push_event_data` method. So the query is inevitable. Couldn't find a way to cache that

Reply:
We can avoid doing count here but rather go for conversation.messages.any? Instead. 
The count will trigger an additional count query, and tbh we don't need the specific value of count here anyways. 

https://twitter.com/nateberkopec/status/1654199212479860768?s=20 
https://www.speedshop.co/2019/01/10/three-activerecord-mistakes.html